.ci/after_success.sh

```diff
@@ -31,6 +31,7 @@ if [ ${TRAVIS_PULL_REQUEST} = false ] && [ ${TRAVIS_BRANCH} = 'master' ]; then
     cp -r doc/html/* koreader_doc/
     pushd koreader_doc
 
+    git add .
```

Comment:
I didn't know that. But what does "newer" mean? The Travis VM is on Ubuntu 12.04 iirc.